mantis-server/mantis-server-agent/src/main/java/io/mantisrx/server/agent/metrics/cgroups/MemorySubsystemProcess.java

```diff
@@ -47,4 +55,19 @@ public void getUsage(UsageBuilder usageBuilder) throws IOException {
             log.warn("stats for memory not found; stats={}", stats);
         }
     }
+
+    private void handleV2(UsageBuilder usageBuilder) throws IOException {
+        Long memLimit = cgroup.getMetric(null, "memory.max");
```

Comment:
This also (in theory) could be the literal value of `max`. but I don't know what `Long memLimit` will be in that case.

Reply:
It's easy to supply your own usage calculator. If cgroup is irrelevant in a given env, I would expect the user to just supply a different class.